src/features/search/pages/Search/Search.tsx

```diff
@@ -84,7 +88,10 @@ export function Search() {
           <InsightsMiddleware />
           <SearchHeader searchInputID={searchInputID} searchView={currentView} />
           {currentView === SearchView.Suggestions ? (
-            <StyledScrollView keyboardShouldPersistTaps="handled">
+            <StyledScrollView
+              keyboardShouldPersistTaps="handled"
+              onScroll={onScroll}
```

Comment:
Ce serait pas mieux d'avoir
```suggestion
              onScroll={Keyboard.dismiss}
```

Reply:
Ah oui si effectivement